### PR TITLE
Adapt Worker/Message API to new events structure

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adapts the DedicatedWorkerGlobalScope, ServiceWorkerContainer, Window[EventHandlers], and Worker APIs to conform to the new events structure.  Part of work for #14578.

### CODE REVIEW

**Review by lucalves — APPROVED** [review]

File: api/ServiceWorkerContainer.json [hunk 1]
```diff
@@ -350,115 +350,6 @@
...
-            }
-          },
-          "status": {
-            "experimental": true,
-            "standard_track": true,
-            "deprecated": false
-          }
-        }
-      },
-      "onmessageerror": {
```
Comment (ddbeck) [inline api/ServiceWorkerContainer.json:403]: We've got `message_event` but no `messageerror_event`. What's the story here?

Also, it seems like the notes for Safari ought to be represented _somewhere_.

File: api/WindowEventHandlers.json [hunk 2]
```diff
@@ -293,106 +293,6 @@
           }
         }
       },
-      "onmessage": {
```
Comment (ddbeck) [inline api/WindowEventHandlers.json:296]: Another case where I'm not sure where this data went. What feature does this map to? Or if it doesn't exist, why isn't it added, based on this?

(My hunch here is that this PR is both demixing and consolidating events, but that's rather difficult to understand and verify from the diff alone.)

**Review by ddbeck — COMMENTED** [review]
A few questions here. Thank you!

File: api/ServiceWorkerContainer.json (hunk 1 above)
Comment (queengooborg) [reply api/ServiceWorkerContainer.json:403]: Oops, my bad -- I've added it now!

File: api/WindowEventHandlers.json (hunk 2 above)
Comment (queengooborg) [reply api/WindowEventHandlers.json:296]: For the `WindowEventHandlers` mixin, they have been merged into `Window`!

**Pull request merged**